src/exercises/day-2/points-polygons.rs

```diff
@@ -84,8 +84,8 @@ impl Polygon {
     }
 
     pub fn length(&self) -> f64 {
-        if self.points.is_empty() {
-            return 0.0;
+        if self.points.len() < 3 {
+            return f64::NAN;
```

Comment:
I would prefer to not do this change: the goal of the exercise is to "trick" people to write a few lines of Rust. It's not a goal to teach people about the corner cases of IEEE 754 floating point arithmetic :smile:  Basically, keep it simple so that people can focus on writing methods and on exploring the standard library.

Reply:
Make sense to me. I just reverted the part.